### DISCUSSION TITLE
[Auto Gluon]Validation Set Size Without Specifying `tuning_data`?

### DISCUSSION THREAD

**Discussion by otakuCandy** [amazon-science/chronos-forecasting#276] [Q&A]
Thank you very much for sharing such wonderful work. When fine-tuning with AutoGluon, if I don't specify the `tuning_data` parameter, how does it determine the size of the validation set behind the scenes?
